openapi/index.yaml

```diff
@@ -541,6 +541,8 @@ definitions:
         $ref: '#/definitions/IsEmailValidated'
       is_test_profile:
         $ref: '#/definitions/IsTestProfile'
+      last_app_version:
+        $ref: '#/definitions/Profile/properties/last_app_version'
```

Comment:
To avoid nested type import we should import the `AppVersion` type in `index.yaml.template`